internal/services/postgresql/docker.go

```diff
@@ -0,0 +1,109 @@
+package postgresql
+
+import (
+	"context"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	"github.com/icinga/icinga-testing/utils"
+	"go.uber.org/zap"
+	"time"
+)
+
+type dockerCreator struct {
+	*rootConnection
+	logger        *zap.Logger
+	client        *client.Client
+	containerId   string
+	containerName string
+}
+
+var _ Creator = (*dockerCreator)(nil)
+
+func NewDockerCreator(logger *zap.Logger, dockerClient *client.Client, containerName string, dockerNetworkId string) *dockerCreator {
+	logger = logger.With(
+		zap.Bool("postgresql", true),
+		zap.String("container-name", containerName),
+	)
+
+	networkName, err := utils.DockerNetworkName(context.Background(), dockerClient, dockerNetworkId)
+	if err != nil {
+		panic(err)
+	}
+
+	dockerImage := "postgres:latest"
```

Comment:
latest?

Reply:
Just what's also done for the other images at the moment:

https://github.com/Icinga/icinga-testing/blob/12cf833da4f33bf595091935b74ef26bdefae9a7/internal/services/icinga2/docker.go#L56
https://github.com/Icinga/icinga-testing/blob/12cf833da4f33bf595091935b74ef26bdefae9a7/internal/services/mysql/docker.go#L35
https://github.com/Icinga/icinga-testing/blob/12cf833da4f33bf595091935b74ef26bdefae9a7/internal/services/redis/docker.go#L52

Making this configurable via environment variables in on my to-do list (not particularly high priority though).